rclpy/rclpy/__init__.py

```diff
@@ -54,3 +54,14 @@ def spin(node):
             executor.spin_once()
     finally:
         executor.shutdown()
+
+
+def spin_until_future_complete(node, future):
```

Comment:
Missing a doc block, but so are many existing functions. It would be nice to add, but not required I guess.

Reply:
Added in cf17f386fd9fb2eb1dd6742add50580db2412414